app/models/chargeback_rate_detail.rb

```diff
@@ -23,7 +23,9 @@ class ChargebackRateDetail < ApplicationRecord
 
   def charge(relevant_fields, consumption)
     result = {}
-    if (relevant_fields & [metric_keys[0], cost_keys[0]]).present?
+
+    if (relevant_fields & [metric_keys[0], cost_keys[0]]).present? ||
+        (metric_keys[0].include?("quota") && consumption.resource.class.name.include?("Container"))
```

Comment:
You're right, removing. this was before i found the right source-metric-group combo in `chargeback_rates.yml`